huntsman_dust/utils.py

```diff
@@ -210,14 +212,11 @@ def mask_galaxy(image,
     Y, X = np.ogrid[:y, :x]
     dist_from_center = np.sqrt((X - a)**2 + (Y - b)**2)
     mask = dist_from_center <= rad_pix
-    masked_img = image.copy()
-    masked_img[(mask > 0)] = 0
-
-    return masked_img, mask
+    return mask
 
 
 def plt_fits(image,
-             wcs,
+             wcs=None,
              figure=None,
```

Comment:
Don't specify `figure=None` do `figure` (and check for all other functions). With `figure=None` I can call the function as `plt_fits()` and it won't fail until it calls `fig = plt.figure(num=figure)` (same for `ax.imshow(image, origin='lower', cmap=cmap, norm=norm)` ). By doing `figure` you're requiring an input. If it optional, then do something like `figure=1` so it defaults to something useful. This style is thorughout your code, so you'll want to move away from if you can.